data/dev/analysis/filters/filter001/rubrics/_filters_rubric_template.yaml

```
@@ -0,0 +1,5 @@
+# Common rubric variables for filter001.
+topology_target: RC low-pass (first-order)
+topology_rejects: high-pass, band-pass, notch, all-pass
+grounded_guidance: Cite R/C elements and nodes that form the filter path.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)